com.unity.multiplayer.mlapi/Runtime/Messaging/RpcBatcher.cs

```diff
@@ -35,6 +35,13 @@ public SendStream()
         // Used to mark longer lengths. Works because we can't have zero-sized messages
         private const byte k_LongLenMarker = 0;
 
+        private readonly NetworkManager m_NetworkManager;
+
+        public RpcBatcher(NetworkManager networkManager)
+        {
+            m_NetworkManager = networkManager;
+        }
+
```

Comment:
@bendoyon Is this part of the merge change from above? If not, then it's extraneous since it adds a private member that is never used in this PR